src/feed/handler.ts

```diff
@@ -1,17 +1,18 @@
 import { Bytes } from '../utils/bytes'
 import { makeContentAddressedChunk } from '../chunk/cac'
+import Long from 'long'
 
 const TopicLength = 32
 
-export function epocId(time: any, level: any): Uint8Array {
-  const result = new Uint8Array(8)
-  result[7] = 31
+export function epocId(time: number, level: number): number[] {
+  const base = Long.fromNumber(time).and(Long.MAX_UNSIGNED_VALUE.shiftLeft(level))
+  const result = base.toBytes(true)
+  result[7] = level
```

Comment:
So does it use only a byte length? then why did you add `long` as a dependency that use 8 bytes? you can use the default bitwise operators on `number` type instead.